x-pack/platform/plugins/shared/security/server/authorization/authorization_service.ts

```diff
@@ -193,26 +190,29 @@ export class AuthorizationService {
         }
 
         if (canRedirectRequest(request)) {
-          const customBrandingValue = await customBranding.getBrandingFor(request, {
-            unauthenticated: false,
-          });
           const next = `${http.basePath.get(request)}${request.url.pathname}${request.url.search}`;
-          const body = renderToString(
-            <ResetSessionPage
-              staticAssets={http.staticAssets}
-              basePath={http.basePath}
-              logoutUrl={http.basePath.prepend(
-                `/api/security/logout?${querystring.stringify({ next })}`
-              )}
-              customBranding={customBrandingValue}
-            />
+
+          const location = http.basePath.prepend(
+            `/security/reset_session?next=${encodeURIComponent(next)}`
           );
 
+          const body = dedent`
+            <html>
+              <head>
+                <title>Elastic</title>
```

Comment:
__question:__ 🚲 how do you feel about something like this to remove "blinking"?
```suggestion
                <title>Elastic</title>
                <style>
                  body { opacity: 0; animation: fade-in 0.5s ease-in 2s forwards; }
                  @keyframes fade-in { to { opacity: 1; } }
                </style>
```

That's what we have without CSS:

https://github.com/user-attachments/assets/ccd82e27-496c-4f66-8b24-c088a016c1c7

And this is what we can get with CSS:

https://github.com/user-attachments/assets/e4f1bf39-95bc-46fd-aaf8-082cfbd93931

Reply:
Yeah, if we feel like it's important enough, we could probably use `nonce` since we fully control CSP for these pages. That being said, such UX improvements are OK if they are cheap to make, otherwise, it's not a big deal.

Ah, and I forgot to mention, we probably want to mirror the refresh directive via `<meta>` in HTML too, just in case.

Reply:
Done in https://github.com/elastic/kibana/pull/238421/commits/ba7ef5462831cfbba738ed88bf490c31eafe68ac